main.js

```diff
@@ -13,4 +20,18 @@ function turnCar(event) {
   }
 }
 
+function forward(event) {
+  var currentX = parseInt(image.style.left);
+  car.location.x = currentX;
+  car.location.x++;
+  image.style.left = car.location.x + 'px';
+}
+
+function moveCar(event) {
+  if (event.key === ' ') {
+    setInterval(forward, 16);
+  }
+}
+
 window.addEventListener('keydown', turnCar);
+window.addEventListener('keydown', moveCar);
```

Comment:
Instead of doing a separate `keydown` handler, you should include this in your existing `keydown` handler